Core/gesture/__init__.py

```diff
@@ -262,20 +296,54 @@ def loop(self):
                 self.hand_exit = True
                 self.last_function_time = time.time()
 
+
             # Move Mouse
             if (len(gesture_cords) > 0):
-                mouse_pos = (max(0,min(1920,int(gesture_cords[8][0]*1920*1.2))),max(0,min(1080,int(gesture_cords[8][1]*1080*1.4))))
+                #print(gesture_cords[8])
+
                 
-                if sys.platform == 'windows':
+                # this line is a beast    
+                #mouse_pos = (max(0,min(self.screen_x,int(gesture_cords[0][0]*self.screen_x*1.2))),max(0,min(self.screen_y,int(gesture_cords[0][1]*self.screen_y*1.4))))
+
+                new_pos = (max(0,min(self.screen_x,int(gesture_cords[0][0]*self.screen_x*1.2))),max(0,min(self.screen_y,int(gesture_cords[0][1]*self.screen_y*1.4))))
+
+
+                if len(self.pos_history) > 0:
```

Comment:
I can make this change if you want but I recommend using a deque for pos_history, set the maxlen and it becomes an automatic queue. Just gotta append and it handles the popping for u.  Look at gesture_history variable for an example.